homeassistant/components/devolo_home_network/binary_sensor.py

```diff
@@ -52,7 +52,7 @@ class DevoloBinarySensorEntityDescription(
         device_class=BinarySensorDeviceClass.PLUG,
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
-        icon="mdi:router-network",
+        translation_key="connected_to_router",
```

Comment:
There are constants for all string you used in the const.py. Shouldn't we reuse them?

Reply:
Found out that all translation keys are set to the key, so we didn't have to declare them explicitly again